tests/blog_api/test_views.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+from django.http import response
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APIRequestFactory
+
+from app.blog.models import BlogPost, HomePageContent
+from app.blog.views import BlogPostDetailView
+from app.config.settings import REST_FRAMEWORK
+
+""" HOME PAGE CONENT """
+
+
+@pytest.mark.django_db
+def test_home_page_content_always_returns_last_or_empty(client):
+    # Assert returns empty string when there is no data
+    response = client.get(reverse("get_home_page_content", kwargs={"pk": 1}))
+    assert response.data["content"] == ""
+    assert response.status_code == status.HTTP_200_OK
+    # Assert returns last object created
+    HomePageContent.objects.create(content="1")
+    HomePageContent.objects.create(content="2")
+    response = client.get(reverse("get_home_page_content", kwargs={"pk": 1}))
+    assert response.data["content"] == "2"
+    assert response.status_code == status.HTTP_200_OK
+
+
+""" BLOG POSTS """
+
+# SetUp
+@pytest.fixture
+def blog_posts():
+    for _ in range(20):
+        BlogPost.objects.create(title=f"title_{_}", content="Lorem")
+
+
+@pytest.fixture
+def factory():
```

Comment:
usually, we put fixtures in a `conftest.py` file, in this case in `tests/blog_api/conftest.py` to keep them separated from the actual tests